packages/agent/src/commands/install.ts

```diff
@@ -0,0 +1,12 @@
+interface InstallCommandOptions {}
+
+/**
+ * Install required tools to run an Agent workflow file. If the argument do not include
+ * any specific tool, it will go through all the steps and will all required tools.
+ */
```

Comment:
```suggestion
/**
 * Install required tools to run an Agent workflow file. If the argument does not include
 * any specific tool, it will go through all the steps and will install all required tools.
 */
```

Reply:
However, I don't grasp what "if the argument does not include any specific tool" means